src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -90,8 +90,11 @@ const ActionsPane = (props: {contentHeight: string}) => {
   const isClusterDiffVisible = useAppSelector(state => state.ui.isClusterDiffVisible);
   const isActionsPaneFooterExpanded = useAppSelector(state => state.ui.isActionsPaneFooterExpanded);
   const kubeconfigPath = useAppSelector(state => state.config.kubeconfigPath);
-  const [key, setKey] = useState('source');
+
+  const [isApplyModalVisible, setIsApplyModalVisible] = useState(false);
   const [isButtonShrinked, setButtonShrinkedState] = useState<boolean>(true);
+  const [key, setKey] = useState('source');
```

Comment:
what "key" is this? could we find a better variable name? - although I don't think this is from your PR but I just noticed it now